src/main/java/com/limechain/network/protocol/grandpa/GrandpaProtocol.java

```diff
@@ -0,0 +1,76 @@
+package com.limechain.network.protocol.grandpa;
+
+import com.limechain.network.encoding.Leb128LengthFrameDecoder;
+import com.limechain.network.encoding.Leb128LengthFrameEncoder;
+import com.limechain.network.protocol.blockannounce.scale.BlockAnnounceHandshake;
+import io.libp2p.core.Stream;
+import io.libp2p.protocol.ProtocolHandler;
+import io.libp2p.protocol.ProtocolMessageHandler;
+import io.netty.buffer.ByteBuf;
+import io.netty.handler.codec.bytes.ByteArrayEncoder;
+import lombok.extern.java.Log;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.concurrent.CompletableFuture;
+
+@Log
+public class GrandpaProtocol extends ProtocolHandler<GrandpaController> {
+    public static final int MAX_HANDSHAKE_SIZE = 1024 * 1024 * 1024;
```

Comment:
Should be 4

Reply:
This was misnamed and is actually the traffic limit of streams we initiate.